pkg/serverless/invocationlifecycle/lifecycle.go

```diff
@@ -46,6 +56,12 @@ func (lp *LifecycleProcessor) OnInvokeEnd(endDetails *InvocationEndDetails) {
 		endExecutionSpan(lp.ProcessTrace, endDetails.RequestID, endDetails.EndTime, endDetails.IsError)
 	}
 
+	if strings.ToLower(os.Getenv("DD_TRACE_ENABLED")) == "true" &&
```

Comment:
since this check is done twice, we might want to mutualize it + use const for those `DD_` strings